charts/base-cluster/values.yaml

```diff
@@ -75,6 +75,7 @@ traefik:
 speedtest:
   enabled: true
   host: speedtest
+  repository: adolfintel/speedtest
```

Comment:
Technically it is not a `repository`? At other places in this file, we use the following scheme:

Example from above:

```yaml
image:
      registry: docker.io
      repository: alpine/helm
      tag: 3.12.1
```

Reply:
I followed this schema. If you have a look at speedtest.yaml from line 29 to 31 you can see the same structure.

Reply:
helm templated